### PR TITLE
Full FR translation for modules

### PR DESCRIPTION
**Pull request by mattnounet**

### CODE REVIEW

File: src/modules/extendedBuilding/i18n/fr_FR.root..json [hunk 1]
```diff
@@ -0,0 +1,62 @@
+{
```
Comment (KBOE2) [inline src/modules/extendedBuilding/i18n/fr_FR.root..json:1]: Whole file is named ..json (two dots)

**Review by KBOE2 — CHANGES_REQUESTED** [review]
Looks fine to me except for the one comment currently.

File: src/modules/extendedBuilding/i18n/fr_FR.root..json (hunk 1 above)
Comment (mattnounet) [reply src/modules/extendedBuilding/i18n/fr_FR.root..json]: Fixed. Thanks

File: src/modules/extendedCallWindow/i18n/fr_FR.js [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+                name: 'Police',
+                vehicleTypes: [8, 18, 19, 20],
+            },
+            {
+                name: 'Nautique',
+                vehicleTypes: [16, 17],
+            },            
+        ],
+        allTab: 'Tout',
+        occupiedTab: 'Follow-up',
```
Comment (Suchty112) [inline src/modules/extendedCallWindow/i18n/fr_FR.js:98]: rename follow-up to this: Poursuivre

File: src/modules/extendedCallWindow/i18n/fr_FR.js [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+            [/^Nautique(s)?$/]: [16, 17],            
+            [/^Choucas$/]: [18],
+            [/^Équipe(s)? cynophile(s)?$/]: [19],
+            [/^Unité(s)? motocycliste(s)?$/]: [20],
+        },
+    },
+    tailoredTabs: {
+        defaultTabs: [
+            {
+                name: 'Véhicles de pompier',
```
Comment (Suchty112) [inline src/modules/extendedCallWindow/i18n/fr_FR.js:77]: Fourgon d’incendie

File: src/modules/extendedCallWindow/i18n/fr_FR.js [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+                vehicleTypes: [0, 1, 2, 12, 13],
+            },
+            {
+                name: 'Autres véhicules d\'incendie',
```
Comment (Suchty112) [inline src/modules/extendedCallWindow/i18n/fr_FR.js:81]: Autres appareils incendie

**Review by Suchty112 — COMMENTED** [review]
these are now the mistakes I have found. These are the "default" tabs according to the missionwindow

**Comment by Suchty112** [comment]
And if you are doing translations again.
in the main fr_FR file
only_alliance_missions: [57, 74],

File: src/modules/asyncButtons/i18n/fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+			"title": "Kick ?"
+		},
+		"set": "Appliquer",
+		"unset": "Retirer"
+	},
+	"memberlistRoles": {
+		"admin": "Administrateur d'alliance",
+		"aufsichtsrat": "Journal de modération",
+		"coadmin": "Coadministrateur d'alliance",
+		"finance": "Chef des finances",
```
Comment (Suchty112) [inline src/modules/asyncButtons/i18n/fr_FR.json:33]: Finances de l’alliance

File: src/modules/asyncButtons/i18n/fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+		"schooling": "Chef de formation",
+		"sprechwunsch_admin": "Chef des transports"
```
Comment (Suchty112) [inline src/modules/asyncButtons/i18n/fr_FR.json:35]: Demande de transport - administrateur

File: src/modules/asyncButtons/i18n/fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+		},
+		"set": "Appliquer",
+		"unset": "Retirer"
+	},
+	"memberlistRoles": {
+		"admin": "Administrateur d'alliance",
+		"aufsichtsrat": "Journal de modération",
+		"coadmin": "Coadministrateur d'alliance",
+		"finance": "Chef des finances",
+		"schooling": "Chef de formation",
```
Comment (Suchty112) [inline src/modules/asyncButtons/i18n/fr_FR.json:34]: Formateur de l’alliance

**Review by Suchty112 — COMMENTED** [review]
and the fix for the memberlistroles ^^

File: src/modules/extendedCallWindow/i18n/fr_FR.js (hunk 2 above)
Comment (mattnounet) [reply src/modules/extendedCallWindow/i18n/fr_FR.js]: Done

File: src/modules/extendedCallWindow/i18n/fr_FR.js (hunk 3 above)
Comment (mattnounet) [reply src/modules/extendedCallWindow/i18n/fr_FR.js]: Done

File: src/modules/extendedCallWindow/i18n/fr_FR.js (hunk 4 above)
Comment (mattnounet) [reply src/modules/extendedCallWindow/i18n/fr_FR.js]: Done

File: src/modules/asyncButtons/i18n/fr_FR.json (hunk 5 above)
Comment (mattnounet) [reply src/modules/asyncButtons/i18n/fr_FR.json]: Done

File: src/modules/asyncButtons/i18n/fr_FR.json (hunk 6 above)
Comment (mattnounet) [reply src/modules/asyncButtons/i18n/fr_FR.json]: Done

File: src/modules/asyncButtons/i18n/fr_FR.json (hunk 7 above)
Comment (mattnounet) [reply src/modules/asyncButtons/i18n/fr_FR.json]: Done

**Review by Suchty112 — APPROVED** [review]

**Review by KBOE2 — APPROVED** [review]

**Pull request merged**